systemdev/testing-locally.rst

```diff
@@ -16,6 +16,179 @@ We only recommend developing packages using a device with Ubuntu Touch installed
 
     Installing packages has a risk of damaging the software on your device, rendering it unusable. If this happens, you can :doc:`reinstall Ubuntu Touch </userguide/install>`.
 
+Building packages in a chroot using sbuild
+------------------------------------------
+
+`sbuild <https://tracker.debian.org/pkg/sbuild>`__ is a tool for building Debian packages from source in an isolated environment using a chroot created by `schroot <https://wiki.debian.org/Schroot>`__. It closely resembles the package build process on the UBports CI system. Using an isolated build environment with a minimal set of pre-installed packages detects any missing build dependencies and sbuild will also detect problems with the resulting packages by running `lintian <https://lintian.debian.org/manual/lintian.html>`__
+
+Prerequisites
+^^^^^^^^^^^^^
+
+A host system running either Debian 11 (Bullseye) or later or Ubuntu 20.04 (Focal Fossa) or later is required. It is possible to set up such a system in a LXD container or VM (not recommended due to performance) on another distribution or operating system.
+
+A LXD container requires the following configuration setting in order to allow debootstrap to use the mknod system call for creating pseudo devices such as `/dev/null` inside a chroot::
+
+    lxc config set <container> security.syscalls.intercept.mknod true
+
+It is assumed that the user who is building packages is allowed to execute commands with superuser privilege using `sudo`.
+
+sbuild uses schroot in order to manage chroots which in turn uses debootstrap for creating the chroot.  The required packages are installed with::
```

Comment:
should we mark all these tools names as code, ie double backticks?: sbuild, schroot, debootstrap. 

(I would suggest to keep chroot without backticks because this is more used as a word to refer to the generic concept rather than the specific implementation)